File: src/pset1/greedy.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
	float i;
	int count = 0;
	do
	{
	printf("How much money is owed?");
	i = GetFloat();
	}
	while(i<=0);

	i *= 100 ;
	int x = roundf(i) ;

	while(x >= 25)
	{
	x -= 25;
	count++ ;
	}
	while(x >= 10)
	{
	x -= 10;
	count++;
	}
	while(x >= 5)
	{
	x -= 5;
	count++ ;
	}
	while(x >= 1)
	{
	x -= 1;
	count++ ;
	}
	printf("%d\n", count) ;
}

File: src/pset6/greedy.py
from cs50 import get_float
count = 0
i = get_float("How much money is owed: ")
if i >= 1:
	i = get_float("How much money is owed: ")
	i *= 100
	x = round(i)
i *= 100
x = round(i)
while(x >= 25):
	x -= 25
	count += 1

while(x >= 10):

	x -= 10
	count += 1

while(x >= 5):

	x -= 5;
	count += 1

while(x >= 1):

	x -= 1;
	count += 1

print(count)
